tests/lib/rules/prefer-presence-queries.test.ts

```diff
@@ -16,38 +17,93 @@ const queryAllByQueries = ALL_QUERIES_METHODS.map(
   (method) => `queryAll${method}`
 );
 
-type RuleValidTestCase = TSESLint.ValidTestCase<[]>;
-type RuleInvalidTestCase = TSESLint.InvalidTestCase<MessageIds, []>;
+type RuleValidTestCase = TSESLint.ValidTestCase<Options>;
+type RuleInvalidTestCase = TSESLint.InvalidTestCase<MessageIds, Options>;
 
 type AssertionFnParams = {
   query: string;
   matcher: string;
   messageId: MessageIds;
   shouldUseScreen?: boolean;
+  assertionType: keyof Options[number];
 };
 
-const getValidAssertion = ({
+const getValidAssertions = ({
   query,
   matcher,
   shouldUseScreen = false,
+  assertionType,
+}: Omit<AssertionFnParams, 'messageId'>): RuleValidTestCase[] => {
+  const finalQuery = shouldUseScreen ? `screen.${query}` : query;
+  const code = `expect(${finalQuery}('Hello'))${matcher}`;
+  return [
+    {
+      code,
+    },
+    {
+      code,
+      options: [
```

Comment:
You are now passing the `options` property to all the test cases involving the presence or absent matchers. Could you add some additional tests not setting any `options` to check that by default both options are enabled?

Reply:
I've actually adapted the `getValidAssertion` method to return an array of valid test cases rather than a single test case, including the case with no options set, the case with neither option set, and the case with just the intended option set. By contrast, `getInvalidAssertion` has been adapted to return an array of invalid test cases, one with no options set, and one with only the intended option set (as having both options disabled cannot result in an invalid test case).

Reply:
Aah sorry, my bad. I missed the case with no options set. All good then!